### PR TITLE
unit tests porting

### LINKED ISSUE #1483: unit test integration
- [x]  assessmentRoleAllowanceTest.ts -> users.test.ts
- [x]  assessmentTest.ts -> assessments.test.ts
- [x]  bigNumberTest.ts -> numbers.test.ts
- [x]  estimationEngineTest.ts -> estimationEngine.test.ts
- [x]  numberInputTest.ts -> numberInput.test.ts
- [x]  odpPasteTest.ts -> handlePaste.test.ts
- [x]  originalDataPointTest.ts -> odps.test.ts
- [x]  <s>package.json</s>
- [x]  relativeDateTest.ts -> dates.test.ts
- [x]  <s>tableMappingTest.ts</s>
- [x]  <s>tableTest.ts</s>
- [x]  <s>validateOriginalDataPointTest.ts</s> (should be discussed)

### PR DESCRIPTION
**Pull request by rfontanarosa**
closes #1483

### CODE REVIEW

File: src/utils/dates.test.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import * as RelativeDate from './dates'
```
Comment (sorja) [inline src/utils/dates.test.ts:1]: Can we import { RelativeDate } ? (not related to PR but nice to have)

File: src/utils/dates.test.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+const i18nMock = {
```
Comment (sorja) [inline src/utils/dates.test.ts:3]: Should we use jest.mock?

File: src/utils/numberInput.test.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import { acceptNextDecimal, acceptNextInteger } from './numberInput'
```
Comment (sorja) [inline src/utils/numberInput.test.ts:1]: Should we move ./numberInput where it is used?
(NumberInput.tsx or similar)

Comment (sorja) [reply src/utils/numberInput.test.ts:1]: It was moved here temporarily when refactoring, I think it needs a new home

**Review by sorja — CHANGES_REQUESTED** [review]
Left some comments, looks good though!

File: src/utils/numberInput.test.ts (hunk 3 above)
Comment (minotogna) [reply src/utils/numberInput.test.ts:1]: Remove number input

**Review by sorja — APPROVED** [review]

**Review by minotogna — APPROVED** [review]

**Pull request merged**